website/src/pages/index.tsx

```diff
@@ -507,86 +508,104 @@ function MainFeatures(): JSX.Element {
-                  Working with Kubernetes
-                </h2>
-                <p className="leading-relaxed text-base list-disc">
-                  <FontAwesomeIcon icon={faRocket} className="text-purple-700 w-3 h-3 mt-1 mr-2" />
-                  Play Kubernetes YAML directly with Podman Engine
-                </p>
-                <p className="leading-relaxed text-base list-disc">
-                  <FontAwesomeIcon icon={faRocket} className="text-purple-700 w-3 h-3 mt-1 mr-2" />
-                  Generate Kubernetes YAML from pods
-                </p>
-                <p className="leading-relaxed text-base list-disc">
-                  <FontAwesomeIcon icon={faRocket} className="text-purple-700 w-3 h-3 mt-1 mr-2" />
-                  <a href="/docs/kubernetes/deploying-a-pod-to-kubernetes">
-                    Deploy to existing Kubernetes environments
-                  </a>
-                </p>
-                <p className="leading-relaxed text-base list-disc">
-                  <FontAwesomeIcon icon={faRocket} className="text-purple-700 w-3 h-3 mt-1 mr-2" />
-                  <a href="/docs/kind">Running Kubernetes on your workstation with Kind and Podman</a>
-                </p>
-              </div>
+        <div className="flex flex-col text-right w-full mb-5">
+          <div className="flex md:flex-row flex-col-reverse items-center space-x-10 xl:space-x-44">
+            <ThemedImage
+              className="py-4 w-full md:w-1/2 mb-10 md:mb-0"
+              alt="Streamline Kubernetes workflows"
+              sources={{
+                light: useBaseUrl('img/features/kubernetes.png'),
+                dark: useBaseUrl('img/features/kubernetes.png'),
+              }}
+            />
+            <div className="lg:grow md:w-1/2 flex flex-col md:items-end md:text-right items-center text-center">
+              <SectionLabel name="Crafted for Kubernetes" />
+              <h1 className="title-font text-3xl leading-[3rem] lg:text-4xl lg:leading-[4rem] font-extrabold text-gray-900 dark:text-white">
+                Streamline Kubernetes workflows
+              </h1>
+              <p className="leading-relaxed py-5 dark:text-gray-300 text-gray-900">
+                Podman Desktop isn't just a container manager; it's built with Kubernetes at its core. This deep
+                integration means seamless workflows for developers. Work with Podman's native Kubernetes support or
+                effortlessly spin up local Kind or Minikube clusters, bridge to remote environments, and manage your
+                Kubernetes deployments directly through Podman Desktop's intuitive graphical interface. Inspect,
+                configure, and orchestrate your pods, services, and deployments with ease, making Kubernetes management
+                both powerful and user-friendly.
+              </p>
+              <ReadTheDocsButton />
```

Comment:
I think it would be better if this button goes to https://podman-desktop.io/docs/kubernetes